content/technotes/2019-05-21-rodev.md

```diff
@@ -0,0 +1,52 @@
+
+{{< tweet 935562495816753153 >}}
+
+Inspired by [`usethis`](https://usethis.r-lib.org/), we've started work on our specific helpers for rOpenSci package authors, [`rodev`](https://docs.ropensci.org/rodev/). In this note, we'll present some of the helpers it contains at the moment, and ask for your feedback as an rOpenSci package author.
+
+### Setup and scope
+
+To use `rodev`, you'll need to install it from GitHub,
+
+```r
+remotes::install_github("ropenscilabs/rodev")
+```
+
+and then, go to the package project you want to test it on. Indeed, `rodev` works on the active project [as defined by `usethis`](https://usethis.r-lib.org/#usage).
+
+At the moment, [`rodev` functions are divided into four categories in the pkgdown website reference](https://docs.ropensci.org/rodev/reference/index.html). In this post, we'll take a different approach, but hope you'll refer to the online reference too!
+
+### Helper for badges
+
+We recommend, and therefore `rodev` facilitates, the use of [repostatus.org badges](https://www.repostatus.org/) via [`use_repostatus_badge()`](https://docs.ropensci.org/rodev/reference/use_repostatus_badge.html), that calls [`usethis::use_badge()`](https://usethis.r-lib.org/reference/badges.html). At a minimum `rodev::use_repostatus_badge("wip")` will paste the Markdown code of the badge to your clipboard, but as a reminder from `usethis` docs "To add badges automatically ensure your badge block starts with a line containing only `<!-- badges: start -->` and ends with a line containing only `<!-- badges: end -->`." To see what statuses are available, you can call `rodev::repostatus_badges()` that'll output a `data.frame`.
+
+Another badge we help authors adding, if their software is under / underwent review, is a peer-review badge, via `rodev::use_review_badge()` that takes the software review issue number as an argument. E.g. the peer-review badge of the [`tradestatistics` package](https://docs.ropensci.org/tradestatistics/) uses the number 274 because it was reviewed in [issue 274](https://github.com/ropensci/onboarding/issues/274).
+
+### One-liner for collaboration friendliness
+
+To foster collaboration-friendliness of repos, we have templates for issues, PRs and CONTRIBUTING.md. Such files need to live in a .github/ folder. Package authors can get our templates by running `rodev::use_ro_github()` and voilà! the package has a .github/ folder ready to be committed and pushed.
+
+### DESCRIPTION tweaking
+
+We've been thinking about asking authors of peer-reviewed packages to add a mention of rOpenSci in the description of their package DESCRIPTION (this sentence is case-sensitive!). We haven't added this to our guidance yet, but if/when we do, `rodev::add_ro_desc()` will add the sentence "This package has been peer-reviewed by rOpenSci (v. `{version}`)." to the description, with `version` automatically replaced by the package current version.
```

Comment:
isn't description in DESCRIPTION Description with a capital "D"?

Reply:
oops, good catch!